### DISCUSSION TITLE
Upgrade dependency eventsource to 2.0.2 - CVE-2022-1650

### DISCUSSION THREAD

**Discussion by cleitonjesusobj** [sockjs/sockjs-client#589] [General]
A vulnerability has been detected in eventsource 1.1.0 ([CVE-2022-1650](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-1650)).
Could you please update to version 2.0.2 that the fix was made?

**Comment by nathanmillar16** [comment]
This has been done: https://github.com/sockjs/sockjs-client/pull/590/

**Reply by cleitonjesusobj** [reply]
Thank you.
